homeassistant/components/velbus/cover.py

```diff
@@ -83,16 +84,28 @@ def current_cover_position(self) -> int | None:
 
     async def async_open_cover(self, **kwargs: Any) -> None:
         """Open the cover."""
-        await self._channel.open()
+        try:
```

Comment:
You can make a wrapper so that you only need to mark the methods to use it:
https://github.com/home-assistant/core/blob/d50b993f64f057cd856c995388c4adcb3d2cad44/homeassistant/components/webostv/media_player.py#L87-L111